src/JSInterop/Microsoft.JSInterop/src/Infrastructure/TaskGenericsUtil.cs

```diff
@@ -23,6 +24,9 @@ public static void SetTaskCompletionSourceException(object taskCompletionSource,
     public static Type GetTaskCompletionSourceResultType(object taskCompletionSource)
         => CreateResultSetter(taskCompletionSource).ResultType;
 
+    [SuppressMessage("AOT",
+        "IL3050:Calling members annotated with 'RequiresDynamicCodeAttribute' may break functionality when AOT compiling.",
+        Justification = "MakeGenericType is AOT safe for reference types.")]
```

Comment:
I don't think either warnings being suppressed in this file are valid. Why are the `Tasks` being used here guaranteed to be of reference types?